src/InputWithLabel.jsx

```diff
@@ -0,0 +1,27 @@
+// /src/InputWithLabel.jsx
+
+import React, { useEffect, useRef } from 'react';
+
+const InputWithLabel = ({ children, value, onChange }) => {
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    inputRef.current.focus();
+  }, []);
```

Comment:
You could add the todoList as a dependency in the array, so when a new todo is added or removed from the todo list, the focus will be applied again to the input.